engine/src/main/java/com/redhat/cloud/notifications/processors/drawer/DrawerProcessor.java

```diff
@@ -178,16 +178,16 @@ public String buildNotificationMessage(Event event) {
             throw new RuntimeException("Drawer notification data transformation failed", e);
         }
 
-        String message = getTemplate()
+        String message = getTemplate(event.getApplicationId(), event.getEventType().getId(), event.getOrgId())
             .data("data", dataAsMap)
-            .render();
+            .render().trim();
 
         return message;
     }
 
     @CacheResult(cacheName = "drawer-template")
-    TemplateInstance getTemplate() {
-        IntegrationTemplate integrationTemplate = templateRepository.findIntegrationTemplate(null, null, DEFAULT, "drawer")
+    TemplateInstance getTemplate(UUID applicationId, UUID eventTypeId, String orgId) {
+        IntegrationTemplate integrationTemplate = templateRepository.findIntegrationTemplate(applicationId, eventTypeId, orgId, ALL, "drawer")
```

Comment:
Can we get rid of `applicationId`?
```suggestion
    TemplateInstance getTemplate(UUID eventTypeId, String orgId) {
        IntegrationTemplate integrationTemplate = templateRepository.findIntegrationTemplate(null, eventTypeId, orgId, ALL, "drawer")
```

Reply:
Wouldn't this code work if `ALL` was replaced with `EVENT_TYPE`?

Reply:
Not if a template is define at an org id level